content/_index.md

```diff
@@ -19,7 +19,7 @@ carousel:
       description: modern applications.
       details:
         - title: Real programming languages.
-          description: Define infrastructure in JavaScript, TypeScript, Python, or Go.
+          description: Define infrastructure in JavaScript, TypeScript, Python, Go, or .NET Core.
```

Comment:
Is there a good reason to be using "Core" all over the place? The next version is dropping "Core". My personal preference would be to use ".NET" without "Core" by default in most places, and only mention "Core" on the installation page about the .NET Core 3.0 SDK requirement. This would be similar to what we do for Python, where we mention just "Python" in most places, not "Python 3" (vs. 2). @CyrusNajmabadi, @mikhailshilkov, @lukehoban, do you have a preference?

Reply:
I agree. The next version is going to be called ".NET 5". I think it was fine to use Core at the launch to build on the newly released .NET Core 3.0, but I'd start cleaning it up.

Reply:
.NET marketing explicitly asked us to use Core in our blog post, and they seem to do the same, so I think until .NET 5 comes out, this is what we're supposed to be doing.

I agree we want to be consistent as .NET 5 becomes increasingly available, but that's a little ways off, isn't it?

Reply:
That said, in this particular case, I think it's not quite right anyway. ".NET Core" isn't a language. Updated to "any .NET language, including C#, F#, and VB." LMK what you think.